pkg/rancher-desktop/components/MarketplaceCard.vue

```diff
@@ -47,25 +147,27 @@ export default {
   border: 1px solid var(--muted);
   border-top: 4px solid var(--muted);
   transition: all 0.2s ease-in-out;
-  padding: 10px;
-
-  &:hover{
-    cursor: pointer;
-    border-color: var(--accent-btn-hover);
-  }
+  padding: 20px;
+  display: flex;
+  flex-direction: column;
 
   .extensions-card {
     display: flex;
-    flex-direction: row;
-    justify-content: space-between;
-    align-items: center;
+    flex-direction: column;
+    gap: 10px;
```

Comment:
I think that the addition of `flex-grow: 1` would be a nice addition for ensuring that the card footer renders in a consistent location.

```suggestion
    flex-grow: 1;
    gap: 10px;
```